spring-rabbit/src/test/java/org/springframework/amqp/rabbit/annotation/EnableRabbitIntegrationTests.java

```diff
@@ -149,6 +155,16 @@ public void testInvalidPojoConversion() throws InterruptedException {
 
 	public static class MyService {
 
+		@RabbitListener(bindings = {
```

Comment:
No reason to make it as an array if you use only one element.
I mean just only for this test-case